org.mwc.debrief.pepys/postgres.ini

```diff
@@ -0,0 +1,7 @@
+[connection]
```

Comment:
This should be `database` - rather than `connection`.  

I guess the format changed at some point.